src/Umbraco.Core/StringExtensions.cs

```diff
@@ -1480,18 +1480,23 @@ public static string ToSafeFileName(this string text, IShortStringHelper shortSt
         }
 
         /// <summary>
-        /// Validates a string matches a time stamp.
+        /// Validates a string matches a cron tab (for length only).
         /// </summary>
-        /// <param name="input">String with timespan representation (in standard timespan format: https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings)</param>
-        /// <returns></returns>
-        public static bool IsValidTimeSpan(this string input)
+        /// <param name="input">String with timespan representation (in cron tab format: https://github.com/atifaziz/NCrontab/wiki/Crontab-Expression)</param>
+        /// <returns>True if string matches a valid cron expression, false if not.</returns>
+        /// <remarks>
+        /// Considering an expression as valid if it's supported by https://github.com/atifaziz/NCrontab/wiki/Crontab-Expression,
+        /// so only 5 or 6 values are expected.
+        /// </remarks>
+        public static bool IsValidCronTab(this string input)
         {
             if (string.IsNullOrEmpty(input))
             {
                 return false;
             }
 
-            return TimeSpan.TryParse(input, out var _);
+            var parts = input.Split(new string[] { " " }, StringSplitOptions.RemoveEmptyEntries);
+            return parts.Length == 5 || parts.Length == 6;
```

Comment:
Not sure if we should use a more advanced validation.
Regex or even try to parse it? Currently "a b c d e f" is value

Reply:
Yes, I was debating this, as the validation I've added it very light at the moment.

Think ideally we'd parse it using the same library that we'll do so to use it in the hosted service, but to do that, we'd need to add a reference to the library to `Umbraco.Core`, or move these validator components out of here and into `Umbraco.Infrastructure`.

Or we write (or [borrow](https://stackoverflow.com/questions/14203122/create-a-regular-expression-for-cron-statement)) our own regex.  Cron expressions can more complex than I'd thought though!

What would you prefer?

Reply:
I would use clean architecture to inject our one ICrontTabParser (living in core) where needed and have the implementation using NCronTab  in infrastructure

like this:
https://github.com/umbraco/Umbraco-CMS/commit/2bee18c30eee0a11744b232f43479509f103b904

I even found an issue with the validation, because the `* * * * * 1` was invalid in NCronTab

Reply:
Nice, yes that looks good to me.  Only thing I noticed was you've got the name `cronTap` rather than `cronTab` in a few places.

Reply:
I fixed the `cronTap` vs  `cronTab` and cherry picked to commit into this pr